tests/Microsoft.Graph.Core.Test/Requests/Middleware/Content/BatchRequestContentTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
```

Comment:
I propose that we stop adding tests to both the .Net Core and .Net framework test projects and only add tests to the .Net Core. Any reason for us to continue to add dup tests?